tests/utils.py

```diff
@@ -0,0 +1,28 @@
+import os
+import subprocess
+
+import requests
+
+HOST = "http://localhost:8080/api"
+TEST_FOLDER = os.path.dirname(os.path.realpath(__file__))
+RESOURCE_FOLDER = "{}/{}".format(TEST_FOLDER, "resources")
+WAIT_PERIOD = 30
+
+
+def get_current_folder(file):
+    return os.path.dirname(os.path.realpath(file))
+
+
+def run_command(command, stdout=subprocess.PIPE):
+    p = subprocess.Popen(command, stdout=stdout, stderr=subprocess.PIPE, shell=True)
+    return p.communicate()
+
+
+def get_cookies():
+    data = {
+        'username': 'admin',  # TODO: get fro config
+        'password': '123'
```

Comment:
please extract HOST, DB_NAME, username, and password to config file. 
see example in application.yml and application-dev.properties in main.

Reply:
DB_NAME is a test parametr and it could be test local. with other agree